curation_concerns-models/app/actors/curation_concerns/generic_file/actor.rb

```diff
@@ -0,0 +1,154 @@
+module CurationConcerns::GenericFile
+  # Actions are decoupled from controller logic so that they may be called from a controller or a background job.
+  class Actor
```

Comment:
Delete this file.  It was already merged into `curation_concerns-models/app/actors/curation_concerns/generic_file_actor.rb`